deploy/rmo-olm-dance/01-cronjob.yaml

```diff
@@ -28,30 +28,23 @@ spec:
               set -euxo pipefail
               NAMESPACE=openshift-route-monitor-operator
 
-              # Check for the status route-monitor-operator.v0.1.700-g504ef15 version:
-              # do OLM dance The resources will be re-created with the latest
-              # version
-              CSV_PHASE=$(oc -n openshift-route-monitor-operator get csv route-monitor-operator.v0.1.700-g504ef15 -ojsonpath='{.status.phase}{"\n"}') || true
-              # This does nothing if the expected CSV is not found.
-              echo "CSV_PHASE is $CSV_PHASE"
-              if [[ -z ${CSV_PHASE} || ${CSV_PHASE} != "Succeeded" ]]; then
-                # Take backup of the files to be recreated (subscription, catalogsource and operatorgroup)
-                oc -n "$NAMESPACE" get subs route-monitor-operator -oyaml > /tmp/01-route-monitor-operator-subs.yaml
-                oc -n "$NAMESPACE" get catsrc route-monitor-operator-registry -oyaml > /tmp/02-route-monitor-operator-registry.yaml
-                oc -n "$NAMESPACE" get og route-monitor-operator-og -oyaml > /tmp/03-route-monitor-operator-og.yaml
+              CSV_FOUND=$(oc -n "$NAMESPACE" get clusterserviceversions.operators.coreos.com -o name | grep route-monitor-operator) || true
```

Comment:
This would constantly be replacing the CSV as it would always be found. We want to target a specific version.